api/src/main/java/com/codeforcommunity/dto/team/TransferOwnershipRequest.java

```diff
@@ -0,0 +1,34 @@
+package com.codeforcommunity.dto.team;
+
+import com.codeforcommunity.dto.ApiDto;
+import com.codeforcommunity.exceptions.HandledException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TransferOwnershipRequest extends ApiDto {
+  private Integer newLeaderId;
+
+  public TransferOwnershipRequest(Integer newLeaderId) {
```

Comment:
❓ `newLeaderUserId` might be more clear

Reply:
Don't just resolve comments without responding or making a change

Reply:
done